src/main/java/com/nerodesk/om/mock/MkFriends.java

```diff
@@ -83,16 +97,19 @@ public boolean leader() throws IOException {
         assert this.dir != null;
         assert this.user != null;
         assert this.label != null;
```

Comment:
@pinaf could we use `@EqualsAndHashCode(of = { "friends" })` and remove unused `dir`, `user` and `label` ?